src/System.Private.Xml/src/System/Xml/Core/RawTextWriter.ttinclude

```diff
@@ -0,0 +1,44 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
+<#
+switch (WriterType)
+{
+    case RawTextWriterType.Utf8:
+    case RawTextWriterType.Encoded:
+        break;
+    default:
+        throw new ArgumentOutOfRangeException(
+            "WriterType", WriterType,
+            "Unknown value for WriterType.");
+}
+#>
```

Comment:
these can be joined together

Reply:
One is a `<#` block and the other is a `<#+` block.